src/pages/dashboard/Dashboard.tsx

```diff
@@ -120,6 +121,7 @@ const Dashboard = () => {
               theme={theme}
               i18n={i18n}
               decorators={decorators}
+              CONFIG={CONFIG}
```

Comment:
This is not necessary, because inside the component we are using the imported CONFIG variable.
Remove this parameter (also from the props type if necessary